podman/tests/integration/test_container_create.py

```diff
@@ -135,6 +135,24 @@ def test_container_mounts(self):
                 )
             )
 
+    def test_container_devices(self):
```

Comment:
can you exec_run in this test and run `mount | grep "/dev/foo"` or something of the like to make sure the device is actually in the container not just in the config?

Reply:
Okay, I've extended test to check whether the device is visible in running container (it's not visible in mounts, so instead I've listed devices and compared with original major/minor numbers)